### ISSUE TITLE
inconsistent naming

### ISSUE THREAD

**Issue by tecknicaltom** [dxa4481/truffleHog#122]
On the main GitHub page for the project, I see the project name spelled as:
* truffleHog
* Truffle Hog
* Trufflehog

It would help to accurately credit the project if the naming were more consistent

**Comment by dxa4481** [comment]
Good point. It's `truffleHog` unless you're calling it from the command line, and [then it's](https://github.com/dxa4481/truffleHog/blob/dev/setup.py#L17) `trufflehog` unless you're [using it with docker](https://hub.docker.com/r/dxa4481/trufflehog/), then it's also `trufflehog`

Truffle Hog is wrong, I'll fix the readme.
